content/blog/2023/09/09/2023-09-09-incremental-build-detection-probe.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Incremental Build Detection Probe
+tags:
+- gsoc
+- gsoc2023
+- healthscore
+- probes
+- plugin
+authors:
+- jagruti
+opengraph:
+  image: /images/gsoc/2023/jagruti/incremental-build-detection-probe.png
+sig:
+- gsoc
+---
+
+== Introduction
+
+`Incremental Build Detection Probe`  was developed as a part of link:/projects/gsoc/2023/projects/add-probes-to-plugin-health-score/[Google summer of Code 2023] for the plugin health scoring system project.
+
+Incremental Builds aids in continuous delivery of the plugin. It deploys only the modules affected by new commits footnote:[https://www.cloudbees.com/jenkins/maven]. Jenkins core and plugin can be deployed incrementally rather than waiting for the developer to manually deploy the release footnote:[https://github.com/jenkinsci/jep/blob/master/jep/305/README.adoc].
+
+This probe identifies the plugins that have configured `incremental-builds`.
+
+== Importance of the Probe
+This probe encourages automation in delivery and saves maintainer from manual deployment.
+
```

Comment:
```suggestion
== Importance of the Probe

This probe encourages delivery automation and saves the maintainer from manual deployment.

```
Is there any context as to why this is important? Saving effort/time is good, but expanding on why this probe is particularly important or helpful in that regard would be good to have.